packages/lib/src/vis/tiles/hooks.ts

```diff
@@ -1,13 +1,32 @@
-import { useThree } from '@react-three/fiber';
+import type { Domain } from '@h5web/shared';
+import { scaleLinear } from '@visx/scale';
 
 import { useVisibleDomains } from '../hooks';
+import { useAxisSystemContext } from '../shared/AxisSystemContext';
 
-export function useDataPerPixel() {
+export function useScaledVisibleDomains(
```

Comment:
Following our discussion, I still kept a hook since it is needed both in `TiledHeatmap` and `TiledLayer`